From cfc28fd05f4d8414659660438aa24faa11fe86e8 Mon Sep 17 00:00:00 2001
From: dengbo11 <dengbo@deepin.org>
Date: Sat, 12 Oct 2024 17:46:16 +0800
Subject: [PATCH] fix(repo): ll-cli search result is abnormal

PackageInfoV2 packageInfoV2Module is not assignment when construction
PackageInfoV2.

Log: fix ll-cli search abnormally
---
 libs/linglong/src/linglong/repo/ostree_repo.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libs/linglong/src/linglong/repo/ostree_repo.cpp b/libs/linglong/src/linglong/repo/ostree_repo.cpp
index 5ae933cc..84ec991a 100644
--- a/libs/linglong/src/linglong/repo/ostree_repo.cpp
+++ b/libs/linglong/src/linglong/repo/ostree_repo.cpp
@@ -1239,6 +1239,7 @@ OSTreeRepo::listRemote(const package::FuzzyReference &fuzzyRef) const noexcept
           .description = item->description,
           .id = item->app_id,
           .kind = item->kind,
+          .packageInfoV2Module = item->module,
           .name = item->name,
           .runtime = item->runtime,
           .size = item->size,
-- 
2.47.0

